### PR TITLE
Move output of stack rm to stdout

### PR DESCRIPTION
**Pull request by FrenchBen**
**- What I did**
Updated the output of `docker stack rm` to be sent to stdout.

Fixes https://github.com/docker/cli/issues/490

**- How I did it**
`s/Err/Out`

**- How to verify it**
With no errors, doing `docker stack rm nginx 1>stdout.log 2>stderr.log` should show all output in `stdout`

**- Description for the changelog**
`docker stack rm` now sends info messages to Stdout

**- A picture of a cute animal (not mandatory but encouraged)**
doge

### CODE REVIEW

**Comment by dnephin** [comment]
Why should it go to stdout?

**Comment by FrenchBen** [comment]
It's not an error?

**Comment by dnephin** [comment]
Ok, makes sense to be consistent with the other remove commands.

The e2e test needs to be fixed as well

File: cli/command/stack/remove_test.go [hunk 1]
```diff
@@ -101,7 +101,13 @@ func TestRemoveStackSkipEmpty(t *testing.T) {
...
 	assert.NoError(t, cmd.Execute())
-	assert.Equal(t, "", fakeCli.OutBuffer().String())
+	expectedList := []string{"Removing service bar_service1",
+		"Removing service bar_service2",
+		"Removing secret bar_secret1",
+		"Removing config bar_config1",
+		"Removing network bar_network1\n",
+	}
+	assert.Equal(t, strings.Join(expectedList, "\n"), fakeCli.OutBuffer().String())
 	assert.Contains(t, fakeCli.ErrBuffer().String(), "Nothing found in stack: foo\n")
```
Comment (dnephin) [inline cli/command/stack/remove_test.go]: hmm, actually this seems weird. Why is it outputing both "nothing found in stack" and that it's removing things?

Not sure how this is passing. Something seems wrong

Comment (dnephin) [reply cli/command/stack/remove_test.go]: Oh, nevermind, I see it now. There are two stacks, one is empty, the other is not.

**Comment by FrenchBen** [comment]
Can you squash on merge?

**Pull request merged**